sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/tracing/TraceTokenClientInterceptor.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2022- SDA SE Open Industry Solutions (https://www.sda.se)
+ *
+ * Use of this source code is governed by an MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ */
+package org.sdase.commons.spring.boot.web.tracing;
+
+import feign.RequestInterceptor;
+import feign.RequestTemplate;
+import java.util.UUID;
+import org.springframework.web.context.request.RequestAttributes;
+import org.springframework.web.context.request.RequestContextHolder;
+
+public class TraceTokenClientInterceptor implements RequestInterceptor {
```

Comment:
How is it applied to clients?

Background:
We need to distinguish requests within the platform (with this filter) and to external APIs (without this filter).

We should document this like for the `AuthenticationPassThroughClientConfiguration`.

Reply:
Currently it simply applies to all clients. We could rework it to only be applied it you actively specify as a configuration for FeignClients

Reply:
It now has to be added manually to the FeignClient configuration